screenpy/speech_tools.py

```diff
@@ -3,12 +3,14 @@
 """
 
 import re
-from typing import Any, Union
+from typing import TypeVar, Union, overload
 
 from screenpy.protocols import Answerable, Describable, Performable, Resolvable
 
+T = TypeVar("T")
 
-def get_additive_description(describable: Union[Describable, Any]) -> str:
+
+def get_additive_description(describable: Union[Describable, T]) -> str:
```

Comment:
👍 much better typing, thank you!

Reply:
I don't know why I didn't see it sooner.  I think there are a couple other places in the package that might benefit from similar update.